README.md

```diff
@@ -40,6 +40,7 @@ Call this function from the release pipeline.
 - The new version will be calculated based on the previous version, the date and the patch flag.
 - The commit will be tagged with the version.
 - If the --patch flag is not provided, a release branch will be created.
+- The --iterateOnScope can be used in a patching scenario and it needs to be paired with the --scopeTag argument. This is particularly useful in a monorepo where you would wand to maintain multiple scope. Example your release pipeline produces the following tag: `release/v1.YearMonthDay.version` and you wish to patch a package using a different tag pattern. You can achieve this by calling `npx date-based-version --patch --scopedBranch=release/<package> --iterateOnScope` and you will get the following tag: `release/<package>/v1.YearMonthDay.version.patch`.
```

Comment:
Why is this flag needed, can it be on always instead?

Reply:
Originally, I thought this could create undesired creation of tags, but now that you made me think of it this is not that big of an issue; I'll make this the default behavior.